projects/ad9081/src/app_config.h

```diff
@@ -73,6 +73,8 @@
 #define AD9081_TX_LOGICAL_LANE_MAPPING	{0, 2, 7, 7, 1, 7, 7, 3}
 #endif
 
+#define AD9081_JRX_TPL_PHASE_ADJUST 0x0c
```

Comment:
idk... i wouldn't create a macro unless 0xc means something and 0xd means something else...
if it's just a field which can take a range of values (say 0x0..0xf) i would not add such a macro, it just makes for more code

Reply:
This value changes depending on the JESD settings modified here. I created a macro here because I wanted to keep the modifications required when switching between configurations in one place.